### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by ViliLipo**
Docker deployment

### CODE REVIEW

File: backend/app/__init__.py [hunk 1]
```diff
@@ -29,15 +31,20 @@
     }
 )
 
-
+# TODO: set cors origin so that docker works
```
Comment (daubaris) [inline backend/app/__init__.py]: Do we have enough time to do that or is it "future plan"?

File: backend/app/__init__.py [hunk 2]
```diff
@@ -50,6 +57,7 @@
 
 # Adds one test object to the database
 try:
+    print("YOLO")
```
Comment (daubaris) [inline backend/app/__init__.py]: Remove... :smile:

File: backend/app/views.py [hunk 3]
```diff
@@ -30,6 +33,11 @@ def bundle():
     return send_from_directory("dist", "dist.bundle.js")
 
 
+@app.route("/main.css")
```
Comment (daubaris) [inline backend/app/views.py]: Why is this needed?

Comment (ViliLipo) [reply backend/app/views.py]: Well it wont serve the styles otherwise. We have no other file server running on the container than flask. That is not the normal way of doing things so it seems odd, usually static content is served with nginx or something through /var/www. :+1:

Comment (daubaris) [reply backend/app/views.py]: Got it :+1:
